packages/bridge-ui/src/components/form/BridgeForm.svelte

```diff
@@ -289,6 +291,7 @@
       on:input={updateAmount}
       class="input input-primary bg-dark-4 input-md md:input-lg w-full focus:ring-0"
       name="amount"
+      bind:this={amountInput}
```

Comment:
So would it not be wise to bind amount here and then use same function to update the input?